casacore/__init__.py

```diff
@@ -1,2 +1,13 @@
-__version__ = "3.5.2"
+__version__ = "3.6.0a1"
```

Comment:
I prefer 3.5.3 (we'll go to 3.6 when we ship with casacore 3.6 which hasn't been released yet)